src/components/layout.js

```diff
@@ -13,6 +13,13 @@ import "./layout.css"
 import JumbotronElement from "./jumbotron"
 import Footer from "./footer";
 
+if (typeof window !== "undefined") {
```

Comment:
Actually i tried running with and without this section of code, and i dont observe any difference in terms of the scrolling effect. I am not entirely sure what is supposed to do for this

Reply:
Can you do one thing; when you remove that part can you restart the server and start it again with `gatsby develop`? :smile:

Reply:
Also do an `npm install` before the restart please since the `package.json` is updated in this pull request.

Reply:
works as intended, for some reason i was thinking the scroll bar was something else, didnt know its a redirection from nav.link to their respective pages. sorry for confusion